engine/timer.go

```diff
@@ -28,9 +29,15 @@ func Timer(stateI interfaces.IState) {
 			// Leaders running with slightly different minutes in test environments.
 			time.Sleep(time.Duration(s.GetTimeOffset().GetTimeMilli()) * time.Millisecond)
 
-			s.TickerQueue() <- -1 // -1 indicated this is real minute cadence
+			if s.LastSyncTime.Nanoseconds() > tenthPeriod*9/10 && s.Leader {
```

Comment:
Is measuring the sync time really what we care about? Maybe we should measure the time since this minute started and claim that has to be 90% of a minute instead of this.